docs/01-basics/01-basic-window.md

```diff
@@ -0,0 +1,286 @@
+Find a location of your choice somewhere on your file system.
```

Comment:
~~Find a location of your choice somewhere on your file system~~
Find a location somewhere on your file system